Building master on Windows (7, 64 bit) results in link errors in drakeLCMSystem

I guess drake003 is a bit behind (stuck?), so this link error snuck in.

```
"C:\Users\Twan\code\drake\pod-build\ALL_BUILD.vcxproj" (default target) (1) ->
"C:\Users\Twan\code\drake\pod-build\drake.vcxproj" (default target) (11) ->
(CustomBuild target) ->
  LINK : fatal error LNK1181: cannot open input file '..\lib\Release\drake_lcmtypes.lib' [C:\Users\Twan\code\drake\drake\pod-build\systems\drakeLCMSystem.vcxproj] [C:\Users\Twan\c
ode\drake\pod-build\drake.vcxproj]
  LINK : fatal error LNK1181: cannot open input file '..\lib\Release\drake_lcmtypes.lib' [C:\Users\Twan\code\drake\drake\pod-build\systems\drakeLCMSystem.vcxproj] [C:\Users\Twan\c
ode\drake\pod-build\drake.vcxproj]
```

--- a/drake/systems/CMakeLists.txt
+++ b/drake/systems/CMakeLists.txt
@@ -5,8 +5,7 @@ endif()
 
 if (LCM_FOUND)
   add_library_with_exports(LIB_NAME drakeLCMSystem SOURCE_FILES LCMSystem.cpp)
-  add_dependencies(drakeLCMSystem lcmtype_agg_hpp)
-  target_link_libraries(drakeLCMSystem drake_lcmtypes)
+  add_dependencies(drakeLCMSystem drake_lcmtypes lcmtype_agg_hpp)
   pods_use_pkg_config_packages(drakeLCMSystem lcm)
 
   drake_install_headers(LCMSystem.h)

